patterns/just-arrived-full-hero.php

```diff
@@ -5,19 +5,20 @@
  * Categories: WooCommerce
  */
 ?>
-<!-- wp:cover {"useFeaturedImage":true,"dimRatio":0,"minHeight":50,"minHeightUnit":"vw","contentPosition":"center center","isDark":false,"align":"full","style":{"spacing":{"padding":{"left":"50vw"}}}} -->
-<div class="wp-block-cover alignfull is-light" style="padding-left:50vw;min-height:50vw"><span aria-hidden="true" class="wp-block-cover__background has-background-dim-0 has-background-dim"></span><div class="wp-block-cover__inner-container"><!-- wp:heading {"style":{"spacing":{"margin":{"top":"0px","right":"0px","bottom":"0px","left":"0px"}},"typography":{"fontStyle":"normal","fontWeight":"700"}},"fontSize":"x-large"} -->
-<h2 class="has-x-large-font-size" id="just-arrived" style="margin-top:0px;margin-right:0px;margin-bottom:0px;margin-left:0px;font-style:normal;font-weight:700"><?php esc_html_e( 'Just arrived', 'woo-gutenberg-products-block' ); ?></h2>
+<!-- wp:cover {"useFeaturedImage":true,"dimRatio":0,"contentPosition":"center right","isDark":false,"align":"full","style":{"spacing":{"padding":{"right":"4em"}}}} -->
+<div class="wp-block-cover alignfull is-light has-custom-content-position is-position-center-right" style="padding-right:4em"><span aria-hidden="true" class="wp-block-cover__background has-background-dim-0 has-background-dim"></span><div class="wp-block-cover__inner-container"><!-- wp:group {"layout":{"type":"constrained"}} -->
+<div class="wp-block-group"><!-- wp:heading {"style":{"typography":{"fontStyle":"normal","fontWeight":"700"}},"fontSize":"x-large","anchor":"just-arrived"} -->
+<h2 class="wp-block-heading has-x-large-font-size" id="just-arrived" style="font-style:normal;font-weight:700">Just arrived</h2>
 <!-- /wp:heading -->
 
 <!-- wp:paragraph {"fontSize":"medium"} -->
-<p class="has-medium-font-size"><?php esc_html_e( 'Our early autumn collection is here.', 'woo-gutenberg-products-block' ); ?></p>
+<p class="has-medium-font-size">Our early autumn collection is here.</p>
```

Comment:
Not sure, but I think that `esc_html_e` is necessary for the translations.

Reply:
So in all the newer patterns, we no longer added translatable text for them. So I just follow the same thing here. I think the reasoning is because these are just examples and the user will never be using these text and will use their own.

Reply:
TIL